src/elexmodel/client.py

```diff
@@ -86,6 +88,18 @@ def _check_input_parameters(
             raise ValueError("handle_unreporting must be either `drop` or `zero`")
         return True
 
+    # These functions collect the conformalization data from a model run. They each produce a dictionary
```

Comment:
would be good to add these comments as docstrings

Reply:
done